wiki/markdown/configuration.md

```diff
@@ -0,0 +1,61 @@
+# Configuration
+
+To be able to execute and use Eiffel-Intelligence Front-end, some configuration need to provided.
+All configurations in made via application.properties file. These properties can also be provided as Java properties arguments as well.
+
+## Eiffel-Intelligence Backend
+
+Eiffel-Intelligence Frontend is dependent that at least one EI Backend instance is up and running.
+It is possible to configure several EI Backend instances.
+
+EI Backebnd instances is configured by setting "ei.backend.instances.list.json.content" property, example:
```

Comment:
back-end misspelled. Highlight the property name in bold to make it more visible.

Reply:
fixed.